### PR TITLE
Issue 3122: Revised - Separate help text for publishing options

### LINKED ISSUE #3122: [UX] separate help text for publishing options
## Describe your issue or idea

After adding the new scheduled publishing feature in Backdrop 1.10 I noticed that help text for two separate items is grouped together, after all items are listed. 
screen_shot_2018-05-15_at_2 07 02_pm

It would be easier to read and understand if the help text for each item were listed immediately after the item itself.

---
PR by @opi: ~https://github.com/backdrop/backdrop/pull/2195~
PR by @klonos ~https://github.com/backdrop/backdrop/pull/3247~
PR by @jenlampton: ~https://github.com/backdrop/backdrop/pull/3613~
PR by @stpaultim (based upon priors) https://github.com/backdrop/backdrop/pull/3742

### PR DESCRIPTION
**Pull request by stpaultim**
Takes prior work by others and removes the "Additional" wrapper

Fixes backdrop/backdrop-issues#3122

Alternative to #3613

### CODE REVIEW

File: core/modules/node/node.pages.inc [hunk 1]
```diff
@@ -382,16 +385,21 @@ function node_form($form, &$form_state, Node $node) {
     $form['options']['status'][NODE_SCHEDULED]['#access'] = FALSE;
   }
 
+  $form['options']['additional'] = array(
+    '#type' => 'markup',
+    '#markup' => $node->nid ? t('<label>Additional options</label>') : t('<label>Additional actions</label>'),
```
Comment (klonos) [inline core/modules/node/node.pages.inc]: We should be avoiding adding HTML tags in translatable strings. The `<label>` tag can be added to the element using `#prefix`/`#suffix`, like so:

```suggestion
    '#prefix' => '<label>',
    '#markup' => $node->nid ? t('Additional options') : t('Additional actions'),
    '#suffix' => '</label>',
```

File: core/modules/node/node.pages.inc [hunk 2]
```diff
@@ -382,18 +385,33 @@ function node_form($form, &$form_state, Node $node) {
     $form['options']['status'][NODE_SCHEDULED]['#access'] = FALSE;
   }
 
+  $form['options']['additional'] = array(
+    // @todo candidate for conversion to '#type' => 'html_tag'.
+    // See: https://github.com/backdrop/backdrop-issues/issues/5223
+    '#type' => 'markup',
+    '#prefix' => '<label id="promote-stiky-label">',
```
Comment (olafgrabienski) [inline core/modules/node/node.pages.inc]: Should be "sticky", not "stiky".

File: core/modules/node/node.pages.inc [hunk 3]
```diff
@@ -382,18 +385,33 @@ function node_form($form, &$form_state, Node $node) {
...
+  );
 
   $form['options']['promote'] = array(
     '#type' => 'checkbox',
-    '#title' => t('Promoted'),
+    '#title' => $node->nid ? t('Promoted') : t('Promote'),
     '#default_value' => $node->promote,
     '#access' => $node_type->settings['promote_enabled'],
+    '#attributes' => array(
+      'aria-labelledby' => array('promote-stiky-label'),
```
Comment (olafgrabienski) [inline core/modules/node/node.pages.inc]: Same here: "sticky", not "stiky".

File: core/modules/node/node.pages.inc [hunk 4]
```diff
@@ -382,18 +385,33 @@ function node_form($form, &$form_state, Node $node) {
...
+    ),
   );
   $form['options']['sticky'] = array(
     '#type' => 'checkbox',
-    '#title' => t('Sticky at top of lists'),
+    '#title' => $node->nid ? t('Sticky at top of lists') : t('Make sticky at top of lists'),
     '#default_value' => $node->sticky,
     '#access' => $node_type->settings['sticky_enabled'],
+    '#attributes' => array(
+      'aria-labelledby' => array('promote-stiky-label'),
```
Comment (olafgrabienski) [inline core/modules/node/node.pages.inc]: One more: "sticky", not "stiky".

**Review by klonos — APPROVED** [review]
LGTM 👍🏼

**Comment by quicksketch** [comment]
Thank you! Merged into 1.x for 1.21.0.

**Pull request merged**